editor/js/libs/codemirror/codemirror.js

```diff
@@ -32,7 +32,7 @@
   var mac_geMountainLion = /Mac OS X 1\d\D([8-9]|\d\d)\D/.test(userAgent);
   var phantom = /PhantomJS/.test(userAgent);
 
-  var ios = safari && (/Mobile\/\w+/.test(userAgent) || navigator.maxTouchPoints > 2);
+  var ios = false;
```

Comment:
The previous line was to detect iOS devices. Now the `ios` flag is always `false`. 

A change like that does not make sense since the entire variable becomes useless.

Reply:
I know, but I was too lazy to delete the whole editor on iOS, but as I already said: the mobile code editor doesn’t work. And since I was too lazy to delete the mobile editor from the code, setting iOS is the easiest and most effective solution.

Reply:
Still not sure about this change. We normally do not modify third-party libs.

Reply:
I totally understand, but still I beg you: I can‘t use a broken code editor and i don‘t have a PC

Reply:
I have tested the current editor and your patched version on an iPad. 

The issue makes it indeed impossible to properly use the editor in dev. Your version works much better.

Um, I wonder if it is possible to solve the issue by upgrading codemirror. The editor's version is [5.63.0](https://github.com/mrdoob/three.js/pull/22568). However the latest version is [5.65.2](https://codemirror.net/). Maybe it fixes the issue?

Reply:
I've just updated to the 5.65.2: https://github.com/mrdoob/three.js/commit/d2b6998b037caad2f4407bab1e6cce5cac642dfb

I don't see any changes that may be related?

Reply:
the latest version of 2019 might work. iOS users would only have to open the desktop site to make the editor work.